tuteria_application_test/users/tests/test_models.py

```diff
@@ -1,6 +1,6 @@
 import os
 from test_plus.test import TestCase
-from unittest.mock import patch
+from mock import patch
```

Comment:
This particular repo was clearly in python 3 but you changed some of the import statements because you were using python 2. please revert.